payloads/v10/monetization.ts

```diff
@@ -0,0 +1,92 @@
+import type { Snowflake } from '../../globals';
+
+export interface APIEntitlement {
```

Comment:
```suggestion
/**
 * https://discord.com/developers/docs/monetization/entitlements#entitlement-object-entitlement-structure
 */
export interface APIEntitlement {
```

Reply:
Missing `guild_id`

Reply:
=> 
- https://github.com/discord/discord-api-docs/pull/6477/

Reply:
It wasn't deleted but rather moved to the bottom of the table

Reply:
ouch yeah